dace/frontend/python/newast.py

```diff
@@ -259,13 +259,21 @@ def _disallow_stmt(visitor, node):
 ###############################################################
 
 
-def _subset_has_indirection(subset):
+def _subset_has_indirection(subset, PVisitor: 'ProgramVisitor' = None):
```

Comment:
Also, since all the uses below use `self`, we should consider making it a method.

Reply:
On the second comment, no (not yet), because the method is also called by `TaskletTransformer.` Eventually, we need to refactor the frontend to have another class, that will contain the methods `_subset_has_indirection`, `add_indirection_subgraph`, `_add_(read/write)_access`, etc. `TaskletTransformer` and `ProgramVisitor` will inherit from this class. However, it is a major change, for which I don't currently have time.